site/README.md

```diff
@@ -1,70 +1,106 @@
 # Helsinki Design System documentation site
 
-Built with [docz](https://www.docz.site/).
+This is the public documentation site of Helsinki Design System ([hds.hel.fi](https://hds.hel.fi/)). It is built with [docz](https://www.docz.site/) which is powered by [Gatsby](https://www.gatsbyjs.com/). Documentation site includes:
+
```

Comment:
pitäiskö lisätä listaan myös "General introduction of HDS"

Reply:
Korjattu.